data-prepper-core/src/main/java/com/amazon/dataprepper/pipeline/server/DataPrepperServer.java

```diff
@@ -40,12 +48,24 @@ public DataPrepperServer(final DataPrepper dataPrepper) {
             throw new RuntimeException("Failed to create server", e);
         }
 
-        server.createContext("/metrics/prometheus", new PrometheusMetricsHandler());
-        server.createContext("/metrics/sys", new PrometheusMetricsHandler(DataPrepper.getSysJVMMeterRegistry()));
+        if (dataPrepperConfiguration.isMeterRegistryTypeConfigured(MeterRegistryType.Prometheus)) {
+            final PrometheusMeterRegistry prometheusMeterRegistryForDataPrepper =
+                    getPrometheusMeterRegistryFromRegistries(Metrics.globalRegistry.getRegistries());
+            server.createContext("/metrics/prometheus", new PrometheusMetricsHandler(prometheusMeterRegistryForDataPrepper));
```

Comment:
Ques: is it possible to scrape from `CloudWatchMeterRegistry`? Is there any outstanding difference? It seems more consistent to me to use the user configured meter registry type for scraping. But that can also happen later.

Reply:
CloudWatchMeterRegistry publishes to CloudWatch - no scrapping possible